cloud-chatops/cron-clone.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# When copying the contents of this file, remember to name the new file using only valid characters [a-zA-Z0-9_-]
+# Fullstops '.' and file extenstions are not permitted
+
+set -ex
+
+rm -rfv $HOME/cloud-docker-images
+
+cd $HOME
+git clone https://github.com/stfc/cloud-docker-images.git
+
+cd $HOME/cloud-docker-images/cloud-chatops
+docker compose up -d
```

Comment:
Do we not need to do a `docker-compose pull` to grab the latest image?

Reply:
Kalibh has a script that checks for any version changes in the docker-compose file and updates it. So when we delete and pull the repo, it has the newest image (I think, if I understood correctly).

Reply:
Since we do not use the `latest` tag on our image versions, a `pull` will only pull the same image that is already running. The idea being that every git clone will point to the "latest" image as the version should be updated in each pull request.